dist.py

```diff
@@ -108,6 +110,15 @@ def install_cuda_opt_library(
     run_command(*command, '--action', 'install', cwd=workdir)
 
 
+def rename_project(src: str, name: str) -> None:
+    """Rename project.name in pyproject.toml."""
+    assert src.endswith('pyproject.toml')
```

Comment:
Adding log here, as I'd like to keep things transparent as the tool often runs in CI environment.

```suggestion
    assert src.endswith('pyproject.toml')
    log(f'Renaming project name to {name} ({src})')
```